cartesi-rollups_versioned_docs/version-1.5/deployment/self-hosted.md

```diff
@@ -75,6 +75,18 @@ The Cartesi rollups node is distributed as a Docker image. Any popular cloud pro
 
 Alternatively, you can use a service like [Fly.io](https://fly.io/) to deploy your application node.
 
+:::note deploying a reader node
+
+A reader node consumes information from the blockchain but does not participate in the validation process. It will not generate any claims. 
+
+To make your node a reader node create or modify a `.cartesi.env` file in your project directory and set:
```

Comment:
A slight change for better readability:

To configure your node as a reader node, create or modify...

Reply:
Done!